translations/en.json

```diff
@@ -278,7 +278,7 @@
 	"custom-networks": "Custom networks",
 	"dapp": "ÐApp",
 	"dapp-would-like-to-connect-wallet": "would like to connect to",
-	"dapps": "ÐApps",
```

Comment:
this label is used in other places so its better to create new label browser and use it in tabs

Reply:
thanks, done!